packages/core-components/api-report.md

```diff
@@ -1339,6 +1360,12 @@ export type TabIconClassKey = 'root';
 // @public (undocumented)
 export function Table<T extends object = {}>(props: TableProps<T>): JSX.Element;
 
+// @public (undocumented)
+export namespace Table {
+  var // (undocumented)
+    tableIcons: Icons;
```

Comment:
We probably want to do `Object.freeze` on https://github.com/backstage/backstage/blob/590442aac0a02e2271ca772a0c851a83f7422d32/packages/core-components/src/components/Table/Table.tsx#L57

Reply:
So that it looks nice in the report.

Reply:
done!